manim/mobject/svg/tex_mobject.py

```diff
@@ -473,7 +479,9 @@ def break_up_tex_strings(self, tex_strings):
                 pieces.extend(re.split(pattern, s))
         else:
             pieces = tex_strings
-        return [p for p in pieces if p]
+        ret = [p for p in pieces if p]
+        print(ret)
+        return ret
```

Comment:
I don't think that this change here is intentional.

Reply:
Yikes! I thought I reverted that selection but I guess I forgot to save changes before staging...